src/test/java/christmas/model/ReservationDateTest.java

```diff
@@ -0,0 +1,30 @@
+package christmas.model;
+
+import christmas.exception.ExceptionMessage;
+import christmas.model.date.ReservationDate;
+import org.junit.jupiter.api.DisplayName;
+import org.junit.jupiter.params.ParameterizedTest;
+import org.junit.jupiter.params.provider.ValueSource;
+
+import static org.assertj.core.api.Assertions.assertThat;
+import static org.assertj.core.api.Assertions.assertThatThrownBy;
+
+public class ReservationDateTest {
+
+    @ParameterizedTest
+    @ValueSource(ints = {1, 7, 24, 31})
+    @DisplayName("예약 일자 객체 테스트")
+    void createReservationDate(int day) {
+        assertThat(new ReservationDate(day))
+                .isInstanceOf(ReservationDate.class);
+    }
+
+    @ParameterizedTest
```

Comment:
정말 사소하지만, 테스트의 이름을 "~~하면 ~~된다." 와 같이 상황을 서술해 주면 더 흐름을 이해하기 쉽다고 생각해요.
그리고, `@ParameterizedTest`를 사용할 때 이름을 설정해 줄 수 있답니다.
```suggestion
    @ParameterizedTest(name = "날짜에 {0}을 입력하면 예외가 발생한다.")
```